new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -0,0 +1,1237 @@
+    logger.info("Check admin service {0} is created in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkServiceExists(adminServerPodName, domainNamespace);
+
+    // Check managed server pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server pod {0} to be ready in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodReady(managedServerPrefix + i, domainUid, domainNamespace);
+    }
+
+    // Check managed server services created
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check managed server service {0} is created in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkServiceExists(managedServerPrefix + i, domainNamespace);
+    }
+
+    // Check configured cluster configuration is available 
+    boolean isServerConfigured = 
+        checkManagedServerConfiguration("config-cluster-server1");
+    assertTrue(isServerConfigured, 
+        "Could not find managed server from configured cluster");
+    logger.info("Found managed server from configured cluster");
+
+    // Check standalone server configuration is available 
+    isServerConfigured = 
+       checkManagedServerConfiguration("standalone-managed");
+    logger.info("Found non-cluster managed server configuration");
+  }
+
+  /**
+   * Stop the Administration server by patching the resource defintion with 
+   *  spec/adminServer/serverStartPolicy set to NEVER.
+   * Make sure that Only the Administration server is stopped. 
+   * Restart the Administration server by patching the resource defintion with 
+   *  spec/adminServer/serverStartPolicy set to IF_NEEDED.
+   * Make sure that Administration server is in RUNNING state. 
+   */
+  @Test
+  @DisplayName("Restart the Administration server with serverStartPolicy")
+  public void testAdminServerRestart() {
+
+    StringBuffer patchStr = null;
+    patchStr = new StringBuffer("[{");
+    patchStr.append("\"op\": \"replace\",")
+        .append(" \"path\": \"/spec/adminServer/serverStartPolicy\",")
+        .append(" \"value\":  \"NEVER\"")
+        .append(" }]");
+    logger.info("AdminServer shutdown patch string: {0}", patchStr);
+
+    patch = new V1Patch(new String(patchStr));
+    boolean crdPatched = assertDoesNotThrow(() ->
+            patchDomainCustomResource(domainUid, domainNamespace, patch, "application/json-patch+json"),
+        "patchDomainCustomResource(adminShutdown) failed");
+
+    assertTrue(crdPatched, "patchDomainCustomResource(adminShutdown) failed");
+   
+    checkPodDeleted(adminServerPodName, domainUid, domainNamespace);
+    logger.info("AdminServer shutdown success");
+
+    // check managed server pods are not affected
+    logger.info("Check managed server pods are not affected");
+    for (int i = 1; i <= replicaCount; i++) {
+      checkPodReady(managedServerPrefix + i, domainUid, domainNamespace);
+    }
```

Comment:
Instead of just verifying the ms pods are in ready state right after admin server pod is deleted save the pods start times before patching for managed servers and verify the managed servers are not restarted with creation time stamps after admin server pod is restarted

Reply:
Added the logic for managed server pod creation timestamp comparison.